constructor/fcp.py

```diff
@@ -250,19 +251,26 @@ def _main(name, version, download_dir, platform, channel_urls=(), channels_remap
         (x['src'] for x in channels_remap),
     ))
 
-    # set conda to be the user's conda (what is in the environment)
-    # for purposese of getting & building environements, rather
-    # than the standalone conda (conda_exe). Fallback to the
-    # standalone, if needed
-    user_conda = os.environ.get('CONDA_EXE', '') or conda_exe
-
     # make the environment, if needed
     if environment_file:
+        # set conda to be the user's conda (what is in the environment)
+        # for purposese of getting & building environements, rather
+        # than the standalone conda (conda_exe). Fallback to the
+        # standalone, if needed
+        user_conda = os.environ.get('CONDA_EXE', '')
+        if not user_conda:
+            if cc_platform == platform:
+                conda_exe
```

Comment:
Woops. This breaks being able to use constructor without having activated conda. I guess you meant ``user_conda = conda_exe``?

Reply:
Yes, I meant `user_conda = conda_exe`. Sorry about the bug.

Reply:
No worries. Fixed in #422.